Replace the version numbers with the one in FreeBSD ports.

Refer: devel/notify-sharp
--- configure.orig	2020-06-21 21:07:36 UTC
+++ configure
@@ -16763,8 +16763,8 @@ fi
 
 WITH_NOTIFY=$with_notify
 if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"notify-sharp\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "notify-sharp") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"notify-sharp-3.0\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "notify-sharp-3.0") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
@@ -16785,12 +16785,12 @@ if test -n "$NOTIFY_SHARP_CFLAGS"; then
     pkg_cv_NOTIFY_SHARP_CFLAGS="$NOTIFY_SHARP_CFLAGS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"notify-sharp\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "notify-sharp") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"notify-sharp-3.0\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "notify-sharp-3.0") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_NOTIFY_SHARP_CFLAGS=`$PKG_CONFIG --cflags "notify-sharp" 2>/dev/null`
+  pkg_cv_NOTIFY_SHARP_CFLAGS=`$PKG_CONFIG --cflags "notify-sharp-3.0" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -16802,12 +16802,12 @@ if test -n "$NOTIFY_SHARP_LIBS"; then
     pkg_cv_NOTIFY_SHARP_LIBS="$NOTIFY_SHARP_LIBS"
  elif test -n "$PKG_CONFIG"; then
     if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"notify-sharp\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "notify-sharp") 2>&5
+    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"notify-sharp-3.0\""; } >&5
+  ($PKG_CONFIG --exists --print-errors "notify-sharp-3.0") 2>&5
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_NOTIFY_SHARP_LIBS=`$PKG_CONFIG --libs "notify-sharp" 2>/dev/null`
+  pkg_cv_NOTIFY_SHARP_LIBS=`$PKG_CONFIG --libs "notify-sharp-3.0" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
@@ -16828,14 +16828,14 @@ else
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        NOTIFY_SHARP_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "notify-sharp" 2>&1`
+	        NOTIFY_SHARP_PKG_ERRORS=`$PKG_CONFIG --short-errors --print-errors --cflags --libs "notify-sharp-3.0" 2>&1`
         else
-	        NOTIFY_SHARP_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "notify-sharp" 2>&1`
+	        NOTIFY_SHARP_PKG_ERRORS=`$PKG_CONFIG --print-errors --cflags --libs "notify-sharp-3.0" 2>&1`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$NOTIFY_SHARP_PKG_ERRORS" >&5
 
-	as_fn_error $? "Package requirements (notify-sharp) were not met:
+	as_fn_error $? "Package requirements (notify-sharp-3.0) were not met:
 
 $NOTIFY_SHARP_PKG_ERRORS
 
